index.html

```diff
@@ -15,43 +15,62 @@
-
-  Can We Visit Planets?
-
-  Never in all their history have men been able truly to conceive of the world as one: a single sphere, a globe, having the qualities of a globe, a round earth in which all the directions eventually meet.
-
-  which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it.
-
-  Let's Go For A Walk
-
-  The Earth was small, light blue, and so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.
-
-  It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.
-
-  Sign Up
-
-  <img src="img/space-walk.jpg" alt="Space walking astronaut">
-
-  Copyright SpaceWalkers 2018
-
+    <section class="nav-head"> 
+    <h1>SpaceWalker</h1>
+
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+        <a href="#">Sign In</a>
+      </nav>
+    </section>
+
+  <div class="container">  
+    <img class="header-img" src="img/main-header.jpg" alt="Space view of Earth">
+
+    <section class="lift-off-text">
+      <h2>Lift Off</h2>
+
+      <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. 
+        That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
+    </section>
+  
+    <div class="image-trio-middle">
+      <img src="img/blast.jpg" alt="Rocket blasting off">
+
+      <img class="second-img" src="img/sky.jpg" alt="Space sky image">
+
+      <img class="third-img" src="img/launch.jpg" alt="Launching rocket">
+    </div>
+  
+    <section class="middle-lower-text-block">
+      <h2>Can We Visit Planets?</h2>
+
+      <p>Never in all their history have men been able truly to conceive of the world as one: a single sphere, a globe, having the qualities of a globe, 
+        a round earth in which all the directions eventually meet.</p>
+      
+      <p class="mid-lower-second-p">which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it.</p>
+    </section>
+  
+    <section class="lower-text-block">
+      <div class="walk-button">    
+        <h2>Let's Go For A Walk</h2>
+
+        <p>The Earth was small, light blue, and so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.</p>
+        <p class="lower-text-block-second-p">It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, 
+          and my thumb blotted out the planet Earth.</p>
+
+        <button>Sign Up</button>
```

Comment:
having the `Sign Up` button width as a % of the screen width doesn't make much sense for widths >500px